.github/workflows/check-markdown-links.yml

```diff
@@ -0,0 +1,28 @@
+name: "Check markdown links"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+
+jobs:
```

Comment:
❌ please set permission on the job level and clear permission on the top level

examples:
https://github.com/gradle/gradle/blob/13697e813e5e18e79735557a84a5b8ad261e66fc/.github/workflows/contributor-pr.yml#L23

https://github.com/gradle/gradle/blob/13697e813e5e18e79735557a84a5b8ad261e66fc/.github/workflows/contributor-pr.yml#L28